expression/expr_to_pb_test.go

```diff
@@ -30,6 +31,14 @@ import (
 	"github.com/pingcap/tipb/go-tipb"
 )
 
+func init() {
+	fpname := "github.com/pingcap/tidb/expression/PanicIfPbCodeUnspecified"
```

Comment:
Why not put it to `TestPanicIfPbCodeUnspecified`?

Reply:
It's should affect all tests.